debian/control

```diff
@@ -7,7 +7,7 @@ Build-Depends: debhelper (>= 9)
 
 Package: mender-client
 Conflicts: mender
-Architecture: armhf
+Architecture: any
```

Comment:
Is this overridden by the `dpkg-buildpackage` flags, or did you include this by mistake?

Reply:
It is the right thing to have for a multi arch package (and it is the same we have in Debian upstream recipe).

Basically it tells apt-get (if we put this on a repo) that the package will be available for any architecture, so then it will pick the one that matches the host architecture

Note that this is different that writing `all`, which means that the package is architecture agnostic (for example and script).

Reply:
If you are curios, ref https://www.debian.org/doc/debian-policy/ch-controlfields.html#architecture